crypto/CMakeLists.txt

```diff
@@ -111,14 +111,7 @@ if (CONFIG_NRF_CC3XX_PLATFORM)
   #
   zephyr_sources(${NRF_CC3XX_PLATFORM_BASE}/src/nrf_cc3xx_platform_abort_zephyr.c)
   zephyr_sources(${NRF_CC3XX_PLATFORM_BASE}/src/nrf_cc3xx_platform_mutex_zephyr.c)
-endif()
 
-if (CONFIG_CC3XX_BACKEND)
-  if(CONFIG_SOC_NRF5340_CPUAPP)
-    set (CC3XX_ARCH cc312)
-  else()
-    set (CC3XX_ARCH cc310)
-  endif()
```

Comment:
I don't see how this change relates to the commit message ?
> crypto: Link mbedcrypto library when cc3xx platform is enabled
>
> -Changed CMakeLists.txt in crypto to always link the mbedcrypto library

if this is a pure cleanup, then please make such cleanups in independent commits to make them easier to review.

Reply:
This is a pure cleanup indeed. I will split it to a different commit then.